tests/providers/sftp/operators/test_sftp.py

```diff
@@ -417,21 +614,48 @@ def test_arg_checking(self):
             pass
         assert task_3.ssh_hook.ssh_conn_id == self.hook.ssh_conn_id
 
+        # when you work with specific files, then you should use *_files arguments
+        task_4 = SFTPOperator(
+            task_id="task_4",
+            ssh_conn_id=TEST_CONN_ID,
+            local_folder="/tmp/dir_for_remote_transfer/from_remote/csv/",
+            remote_filepath=[
+                "/tmp/transfer_file/remote/put_files_file1.txt",
+                "/tmp/transfer_file/remote/put_files_file2.txt",
+            ],
+            operation=SFTPOperation.GET,
+            create_intermediate_dirs=True,
+        )
+        try:
+            task_4.execute(None)
+        except Exception:
+            pass
+        assert task_4.local_filepath is None
+
     def delete_local_resource(self):
         if os.path.exists(self.test_local_filepath):
-            os.remove(self.test_local_filepath)
+            if os.path.isdir(self.test_local_filepath):
+                shutil.rmtree(self.test_local_filepath)
+            else:
+                os.remove(self.test_local_filepath)
         if os.path.exists(self.test_local_filepath_int_dir):
-            os.remove(self.test_local_filepath_int_dir)
+            if os.path.isdir(self.test_local_filepath_int_dir):
+                shutil.rmtree(self.test_local_filepath_int_dir)
+            else:
+                os.remove(self.test_local_filepath_int_dir)
         if os.path.exists(self.test_local_dir):
-            os.rmdir(self.test_local_dir)
+            if os.path.isdir(self.test_local_dir):
+                shutil.rmtree(self.test_local_dir)
+            else:
+                os.remove(self.test_local_dir)
 
     def delete_remote_resource(self):
         if os.path.exists(self.test_remote_filepath):
             # check the remote file content
             remove_file_task = SSHOperator(
                 task_id="test_check_file",
                 ssh_hook=self.hook,
-                command=f"rm {self.test_remote_filepath}",
+                command=f"rm -rf {self.test_remote_filepath}",
```

Comment:
```suggestion
                command=f"rm -r {self.test_remote_filepath}",
```

But is this (and the if-else things above) needed? I don’t see anywhere a `filepath` points a directory, or a `dir` points to a file. The usages seem pretty consistent.